sdk/spring/azure-spring-integration-servicebus/src/main/java/com/azure/spring/integration/servicebus/converter/ServiceBusMessageConverter.java

```diff
@@ -73,32 +77,56 @@ protected ServiceBusMessage fromByte(byte[] payload) {
     @Override
     protected void setCustomHeaders(MessageHeaders headers, ServiceBusMessage message) {
 
+        Map<String, Object> copyHeaders = new HashMap<String, Object>();
```

Comment:
We should come up with a more descriptive name here.

Reply:
Change for headersForApplicationProperties, is it OK?